src/pointers.cpp

```diff
@@ -418,6 +418,12 @@ namespace big
 			m_invalid_mods_crash_detour = ptr.add(1).rip().as<PVOID>();
 		});
 
+		// Send Chat Ptr
+		main_batch.add("SCP", "83 7E 1C 01 48 8B 3D", [this](memory::handle ptr)
```

Comment:
think this is a bad sig

Reply:
yep, could not update the whole thing correctly yet, as described above

Reply:
disregard

Reply:
pretty good, works!